### DISCUSSION TITLE
关闭播放窗口时会出现2-3秒卡顿

### DISCUSSION THREAD

**Discussion by zfekin** [dyphire/mpv-config#27] [General]
关闭播放窗口时会出现2-3秒卡顿然后才关闭

**Comment by dyphire** [comment]
无法复现，只能猜测可能和 [simple-mpv-webui](https://github.com/dyphire/mpv-config/tree/master/scripts/simple-mpv-webui) 脚本组有关。它实现的webui功能需要通过ipc与外部通信，有可能会影响到播放器关闭时的行为。尝试禁用webui功能看是否有改善
https://github.com/dyphire/mpv-config/blob/41f8d9d9aad6ee66ebfa3e31bee89625628e40b0/script-opts/webui.conf#L4-L5

**Reply by zfekin** [reply]
试过了没有效果

**Reply by dyphire** [reply]
那就没头绪了，命令行测试`mpv --no-config path`看下是否消失，`path`为具体的视频路径，带空格等特殊符号的话需用''包含

**Reply by zfekin** [reply]
716的版本就没这个问题了

**Reply by dyphire** [reply]
那可能是和mpv版本有关
